src/main/log/Resources/modules/administration/logs/store/reducer.js

```diff
@@ -0,0 +1,21 @@
+
+import {combineReducers} from '#/main/app/store/reducer'
+import {makeListReducer} from '#/main/app/content/list/store/reducer'
+
+import {selectors} from '#/main/log/administration/logs/store/selectors'
+
+const reducer = combineReducers({
+  securityLogs: makeListReducer(selectors.LIST_NAME, {
+    sortBy: {property: 'date', direction: -1}
+  }),
+  messageLogs: makeListReducer(selectors.MESSAGE_NAME, {
+    sortBy: {property: 'date', direction: -1}
+  }),
+  functionalLogs: makeListReducer(selectors.FUNCTIONAL_NAME, {
+    sortBy: {property: 'date', direction: -1}
+  })
```

Comment:
Your store is already mounted in a `logs` key in the app store. So naming can be simplified with `security`, `message`, `functional`